src/SprykerEco/Zed/Payone/Business/Api/Request/Container/KlarnaAuthorizationContainer.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Api\Request\Container;
+
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\AbstractAuthorizationContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\BusinessContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\RedirectContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\GenericPayment\PaydataContainer;
+
+class KlarnaAuthorizationContainer extends AbstractAuthorizationContainer
+{
+    /**
+     * @var string
+     */
+    protected $request = PayoneApiConstants::REQUEST_TYPE_AUTHORIZATION;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\BusinessContainer|null
+     */
+    protected $business;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Request\Container\GenericPayment\PaydataContainer|null
+     */
+    protected $paydata;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\RedirectContainer|null
+     */
+    protected $redirect;
+
+    /**
+     * @var string|null
+     */
+    protected $financingtype;
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\BusinessContainer $business
+     *
+     * @return $this
+     */
+    public function setBusiness(BusinessContainer $business)
+    {
+        $this->business = $business;
+
+        return $this;
```

Comment:
* Do the same for others setter or remove it.

Reply:
Removed